translations/admin.en.yaml

```diff
@@ -988,3 +988,5 @@ allow_asset_inline_download: Allow inline download
 system_appearance_settings: 'Appearance & Branding'
 male: Male
 female: Female
+pdf_js_unsafe: This Pdf file contains JavaScript. If you want to view it, please download and open it in your local Pdf viewer.
+pdf_scan_in_progress: 'Preview not available: Pdf is being scanned. This may take a while.'
```

Comment:
```suggestion
pdf_js_unsafe: This PDF file contains JavaScript. If you want to view it, please download and open it in your local PDF viewer.
pdf_scan_in_progress: 'Preview not available: PDF is being scanned. This may take a while.'
```